src/io/flutter/view/FlutterView.java

```diff
@@ -234,6 +233,7 @@ private void addInspector(FlutterApp app, @Nullable InspectorService inspectorSe
       }
 
       addPerformanceTab(runnerTabs, app, !hasInspectorService);
+      runnerTabs.setSelectionChangeHandler(this::onTabSelectionChange);
```

Comment:
You may have answered this from Jacob's review - so I apologize in advance - but why are we registering this here, when we're likely already `onTabSelectionChange()` is (I assume) already being called on tab changes?

Reply:
This registration used to happen above (line 222) and used to only apply if `hasInspectorService`; Jacob pointed out that this was a bug, so I fixed it.  As for why we're explicitly registering at all, I think we need to set the handler explicitly for the runnerTabs or we won't get the changes.  @jacob314 can maybe speak to that though.

Reply:
Gotcha; sounds good (when I originally read this, it seemed like we were doing the selection change listening twice)